spring-pulsar-spring-boot-autoconfigure/src/test/java/org/springframework/pulsar/autoconfigure/SpringPulsarBootAppSanityTests.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.pulsar.autoconfigure;
+
+import org.junit.jupiter.api.Test;
+
+import org.springframework.beans.factory.ObjectProvider;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringBootConfiguration;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.pulsar.autoconfigure.SpringPulsarBootAppSanityTests.SpringPulsarBootTestApp;
+import org.springframework.pulsar.core.PulsarTemplate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Sanity tests to ensure that {@code Spring Pulsar} can be auto-configured into a Spring Boot
+ * application.
+ *
+ * @author Chris Bono
+ */
+@SpringBootTest(classes = SpringPulsarBootTestApp.class)
+public class SpringPulsarBootAppSanityTests extends AbstractContainerBaseTests {
```

Comment:
This make sure if the auto-config is off again that we catch it up front. I only caught this because I manually ran the sample apps after making the milestone downgrade.